### PR TITLE
fix config function

### PR DESCRIPTION
**Pull request by anassohail99**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/a51-finance-v3 exports TVL: 
        
 
 
    polygon-staking           65.80 k
    staking                   65.80 k
    arbitrum                  4.18 k
    optimism                  3.37 k
    blast                     742
    base                      578
    manta                     165
    bsc                       13
    polygon                   12
    scroll                    8
    
    total                    9.07 k

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
is there any code change here that is not formatting related?

**Comment by anassohail99** [comment]
Yes, base chain was only fetching the first dex tvl and leaving the second one due to promise, so I changed promise.all to a for loop inside

**Comment by anassohail99** [comment]
On the defillama inteface it wasn't showing the merge tvl on base chain.

**Comment by anassohail99** [comment]
oh ok @g1nt0ki i got it now, my bad it was i thought the fixed that you did was merged onto main but that wasn't you were just mentioning me to the code right?

**Comment by llamatester** [comment]
The adapter at projects/a51-finance-v3 exports TVL: 
        
 
 
    polygon-staking           65.89 k
    staking                   65.89 k
    arbitrum                  4.18 k
    optimism                  3.38 k
    blast                     744
    base                      580
    manta                     165
    bsc                       13
    polygon                   12
    scroll                    8
    
    total                    9.09 k

File: projects/a51-finance-v3/index.js [hunk 1]
```diff
@@ -1,70 +1,154 @@
...
-
-        const pools = []
-        const reservesCalls = []
-
-        const strategyDetails = await api.multiCall({ abi: getStrategiesDetails, target, calls: strategies, })
-
-        strategyDetails.forEach(({ key: { pool, tickLower, tickUpper }, account }) => {
-          pools.push(pool)
+      for (const config of configs) {
+        await addConfigTvl(config);
```
Comment (g1nt0ki) [inline projects/a51-finance-v3/index.js:88]: I think `await Promise.all()` does the same thing?

Comment (anassohail99) [reply projects/a51-finance-v3/index.js:88]: Yes, but with your change previously the interface wasn't showing the correct tvl for base do you know the reason for that?

Comment (g1nt0ki) [reply projects/a51-finance-v3/index.js:88]: I think there is a bug/cache issue in the UI. Give it 1-2 days, it usually fixes itself

Comment (anassohail99) [reply projects/a51-finance-v3/index.js:88]: Alright, thanks I'm closing this pr then

**Pull request closed without merging**